yt_dlp/extractor/bravotv.py

```diff
@@ -79,12 +86,38 @@ def _real_extract(self, url):
                 'episode_number': int_or_none(metadata.get('episode_num')),
             })
             query['switch'] = 'progressive'
+
+        tp_url = 'http://link.theplatform.com/s/%s/%s' % (account_pid, tp_path)
+
+        tp_metadata = self._download_json(
+            update_url_query(tp_url, {'format': 'preview'}),
+            display_id, fatal=False)
+        if tp_metadata:
+            info.update({
+                'title': tp_metadata.get('title'),
+                'description': tp_metadata.get('description'),
+                'duration': float_or_none(tp_metadata.get('duration'), 1000),
+                'season_number': (
+                    int_or_none(tp_metadata.get('pl1$seasonNumber'))
+                    or int_or_none(tp_metadata.get('nbcu$seasonNumber'))),
+                'episode_number': (
+                    int_or_none(tp_metadata.get('pl1$episodeNumber'))
+                    or int_or_none(tp_metadata.get('nbcu$episodeNumber'))),
+                # For some reason the series is sometimes wrapped into a single element array.
+                'series': try_get(
+                    tp_metadata.get('pl1$show') or tp_metadata.get('nbcu$show'),
```

Comment:
```suggestion
                    dict_get(tp_metadata, ('pl1$show', 'nbcu$show')),
```
OR (with the latest master)
```suggestion
                    traverse_obj(tp_metadata, ('pl1$show',), ('nbcu$show',)),
```

Same for `episode`, `season_number` and `episode_number`

Reply:
Ugh `dict_get` was exactly what I was looking for. I hadn't touched `youtube_dl` in quite a few months and only vaguely recalled it existing. My quick searching in the class didn't uncover it.

Anyway, applied it everywhere I could.